app/components/agenda/agendaitem/agendaitem-postponed.hbs

```diff
@@ -0,0 +1,54 @@
+<div class="auk-alert auk-alert--warning">
+  <div class="auk-alert__body">
+    <Auk::Badge @icon="alert-triangle" @skin={{"warning"}} />
+    <div class="auk-alert__text">
+      <p class="auk-u-text-pre-line auk-alert__message">
+        {{t "postponed-item"}}
+      </p>
+    </div>
+  </div>
+  {{#if @meeting.internalDecisionPublicationActivity.startDate}}
+    {{#if this.canPropose}}
+      <Auk::DropdownMenu @skin="primary" @label={{t "propose-again"}} as |Menu|>
+        {{#if this.loadProposableMeetings.isIdle}}
```

Comment:
Is er een specifieke reden om met `isIdle` te werken ipv loading state te tonen bij `isRunning` zoals we doorgaans doen?

Reply:
`isIdle` is op zich hetzelfde als `not isRunning` dacht ik.